web/src/gql/queries/workspace.ts

```diff
@@ -64,7 +64,46 @@ export const UPDATE_MEMBER_OF_WORKSPACE = gql`
 
 export const REMOVE_MEMBER_FROM_WORKSPACE = gql`
   mutation RemoveMemberFromWorkspace($workspaceId: ID!, $userId: ID!) {
-    removeMemberFromWorkspace(input: { workspaceId: $workspaceId, userId: $userId }) {
+    removeUserFromWorkspace(input: { workspaceId: $workspaceId, userId: $userId }) {
```

Comment:
Can we keep language consistent with the backend by updating the mutation name as well as the variable name to be User instead of Member?

Reply:
@KaWaite , this is not a part of the PR but I did these temporary changes to not break the system.
We had one type of member and then changed it to user/integration on the backend.
Old members are now users.
The whole member section needs refactoring (queries/hooks) because of the backend edit, and it's not part of the integration PR.